src/components/Trade/hooks/useTradeQuoteService.tsx

```diff
@@ -159,8 +159,8 @@ export const useTradeQuoteService = () => {
     buyAsset,
     buyTradeAsset,
     receiveAddress,
+    sellAccountMetadata?.accountType,
```

Comment:
```suggestion
    sellAccountMetadata,
```

want to react on bip44params too